src/proposalBuilder/MinionTributeTransfer.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Flex } from '@chakra-ui/react';
+
+import { PropCardTransfer } from './proposalBriefPrimitives';
+import { ParaMd } from '../components/typography';
+import MediaBox from '../components/mediaBox';
+
+const MinionTributeTransfer = ({ proposal = {} }) => {
+  return (
+    <Flex justify='space-between' alignItems='flex-start'>
+      <PropCardTransfer
+        customUI={
+          <Flex alignItems='center'>
+            <ParaMd mr='2'>For 1 NFT</ParaMd>
+          </Flex>
+        }
+        incoming
+      />
+
+      {proposal?.link && <MediaBox link={proposal.link} width='10%' />}
```

Comment:
Would the media box keep the same card size?

Reply:
oh maybe not. how can we enforce that? Just going on figma design here that display an nft image.